### ISSUE TITLE
Karabiner Elements with International letters (Hungarian) on a Japanese Mac keyboard

### ISSUE THREAD

**Issue by pallasz** [tekezo/Karabiner-Elements#1044]

**Comment by pallasz** [comment]
Hello everybody, 

I've got a big issue with my Japanese Mac. I would like to use with Hungarian keys, but I can't transform with the Karabiner Elements. In the app, I can't find the Hungarian letters. 
In MacOs Sierra I try to use with the Hungarian Qwerty keyboard template, but the is no "0", there is "ö", there is no "í", but there is some unused keys.
I tried many times for transform individual keys to the "right" keys, but I felt again and again.
